### PR TITLE
Free45 flow update

### PR DESCRIPTION
**Pull request by sripathi-muralitharan**
This PR updates the free45 flow and includes the BlackParrot unicore (RC5) toplevel. 

This flow is up to date and tested upto post synth simulations . Future PRs will complete the flow and add even a quad core toplevel for the same release candidate. 

This PR depends on a few changes in the BSG_45 repo - 
1. Updates to folder names
2. Marking SDFFs as don't use

### CODE REVIEW

**Comment by dpetrisko** [comment]
Does this obselete #5 ?

File: free45/toplevels/bsg_ac_black_parrot_single_core_rc5/tcl/common/clock_variables.tcl [hunk 1]
```diff
@@ -0,0 +1,21 @@
+puts "BSG-info: Running script [info script]\n"
```
Comment (dpetrisko) [inline free45/toplevels/bsg_ac_black_parrot_single_core_rc5/tcl/common/clock_variables.tcl]: You can delete the tcl/common folder here, it's not used in this flow

**Comment by dpetrisko** [comment]
For the unsupported testing corners, can you put an $error("<corner> is not currently supported") in the makefile?

**Comment by dpetrisko** [comment]
Other than those two minor things, I think it looks good to go

**Comment by sripathi-muralitharan** [comment]
It doesn't make #5 obsolete if you don't want it to xP I don't see the merit to going back to RC3 flow to fix it now that we have a RC5 flow. But I am not sure if there is some sort of bookkeeping that might come in handy. 

I'll get the rest of the changes in soon

File: free45/toplevels/bsg_ac_black_parrot_single_core_rc5/testing/post_apr_ff/Makefile [hunk 2]
```diff
@@ -4,6 +4,8 @@
 # overriden by the chip repo if this makefile is called from the chip
 # infrastructure.
 
+$(error "Flow unspported! Use at your own peril!")
```
Comment (dpetrisko) [inline free45/toplevels/bsg_ac_black_parrot_single_core_rc5/testing/post_apr_ff/Makefile:7]: :)

**Review by dpetrisko — APPROVED** [review]
LGTM!

**Pull request merged**